.github/workflows/ui-cicd.yml

```
@@ -31,7 +31,7 @@ jobs:
       image-name: 'simplify9/bitweenui'
       helm-set-values: 'image.repo=docker.io/simplify9,ingress.enabled=true,ingress.host=app-dev.bitween.io,ingress.tlsSecret=bitween-tls,ingress.annotations.cert-manager\.io/cluster-issuer=letsencrypt-nginx'
     secrets:
-      registry-username: ${{ secrets.S9_REGISTRY_USERNAME }}
-      registry-password: ${{ secrets.S9_REGISTRY_TOKEN }}
+      registry-username: ${{ secrets.github.actor }}
```

actionlint (1.7.11)

[error] receiver of object dereference "actor" must be type of object but got "string"

(expression)